THIRD_PARTY_README

```diff
@@ -3043,14 +3043,14 @@ performance, or use of this material.
 
 -------------------------------------------------------------------------------
 
-%% This notice is provided with respect to zlib v1.2.13, which may be included 
+%% This notice is provided with respect to zlib v1.3.1, which may be included 
 with JRE 8, JDK 8, and OpenJDK 8.
 
 --- begin of LICENSE ---
 
-  version 1.2.13, October 13th, 2022
+  version 1.3.1, October 13th, 2022
```

Comment:
1.3.1 got released on January 22, 2024. We should use that date.

Reply:
Fix pushed.